### PR TITLE
feat: add coverage adapter overwriting the visit method

### PR DESCRIPTION
**Pull request by SebasthianSalpozz**
US: CS-004 [SP-3] Coverage Adapter
https://app.clickup.com/t/868at9fzm

What I do ?
I work on the Coverage adapter with a function to visit method using the injecter that already exist in our project all with tests

### CODE REVIEW

File: src/main/java/edu/usb/argosinstrumentation/CoverageAdapter/CoverageAdapter.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package edu.usb.argosinstrumentation.CoverageAdapter;
+
+import org.objectweb.asm.ClassVisitor;
+import org.objectweb.asm.MethodVisitor;
+import org.objectweb.asm.Opcodes;
+
+public class CoverageAdapter extends ClassVisitor implements Opcodes {
+    String className;
```
Comment (alex-paca-jalafund) [inline src/main/java/edu/usb/argosinstrumentation/CoverageAdapter/CoverageAdapter.java]: please add private its an atribute.

**Review by alex-paca-jalafund — CHANGES_REQUESTED** [review]

File: src/main/java/edu/usb/argosinstrumentation/CoverageAdapter/CoverageAdapter.java (hunk 1 above)
Comment (SebasthianSalpozz) [reply src/main/java/edu/usb/argosinstrumentation/CoverageAdapter/CoverageAdapter.java]: Sure Also I will use a getter to obtain this atribute :D

**Review by alex-paca-jalafund — APPROVED** [review]
Nice work.

**Review by alex-paca-jalafund — COMMENTED** [review]
Pls add description to the pull request. something like what did you did, why...

**Pull request merged**